backend/corpora/common/corpora_orm.py

```diff
@@ -239,12 +240,16 @@ class DbCollection(Base, AuditMixin, TimestampMixin):
     data_submission_policy_version = Column(StrippedString, nullable=True)
     tombstone = Column(Boolean, default=False, nullable=False)
     publisher_metadata = Column(JSON, nullable=True)
+    revision_of = Column(String, ForeignKey(id), nullable=True)
```

Comment:
```suggestion
    revision_of = Column(String, ForeignKey("project.id"), nullable=True)
```
then you can remove the `id` definition

Reply:
ah, clever